cli/tests/integration_tests.rs

```diff
@@ -320,6 +320,12 @@ itest!(_036_import_map_fetch {
   output: "036_import_map_fetch.out",
 });
 
+itest!(_037_fetch_multiple {
+  args: "fetch --reload fetch/test.ts fetch/other.ts",
+  check_stderr: true,
+  output: "037_fetch_multiple.out",
+});
```

Comment:
Oooops, it needs explicit `http_server: true` as well, then it should work 👍

Reply:
ah! just a second 😅

Reply:
done!